crates/ruma-client-api/src/discovery/discover_support.rs

```diff
@@ -112,6 +112,13 @@ pub enum ContactRole {
     /// A role intended for sensitive requests.
     Security,
 
+    /// A role for moderation-related queries according to MSC4121.
```

Comment:
Could you add a link to MSC4121 please (the PR, not the rendered text).

Reply:
I'm guessing I do that in Markdown?

Reply:
Yes, that's good.